_includes/guide/try_the_api.html

```diff
@@ -74,7 +74,6 @@ <h3>
 	Sample cURL Command to Submit Credentials for an Access Token
 </h3>
 <pre><code>curl -d '' -X POST 'https://sandbox.bcda.cms.gov/auth/token' \
-
 	--user 2462c96b-6427-4efb-aed7-118e20c2e997:8e87f0ebc50d10f1bc9734329a9900179b84ccd39e4d0920b905cc359cf6e94a6e760bbe3a0890c7 \
```

Comment:
@greg-pf it's hard to tell in the github UI; is there a tab before `--user` and before the next `-H`? if so, could we also remove those? 😺

Reply:
would the tabs also need to be removed in [the build instructions](https://github.com/CMSgov/bcda-static-site/blob/deac7639b912dc016e30171a34d990a637be1ff7/_includes/build/access_token.html#L46-L48)?

Reply:
@laurenkrugen-navapbc I confirmed that the command as formatted in the build instructions can be executed with the tabs before `--user` and `-H`. The updated command in this PR is an exact match to that one